.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
-max-complexity = 10
-max-line-length = 88
-extend-ignore = E203,C901,E501
+max-complexity = 101
+max-line-length = 180
```

Comment:
This one should be pretty easy to fix, right? Can't we use black to enforce 88?

Reply:
Yes but black will not autofix all long lines.  For example, lines that are in triple quoted strings.

Reply:
Ah, I remember @targos once mentioned that to me and I volunteered to fix it but didn't actually do it, so I guess that is my fault. I'll try to look into it again.

Reply:
Running __black__ only reformats two files and only fixes 1 long line.  This leaves 62 long lines that black will not fix.

These are nice tasks of first committers or interns.

Reply:
I made issues #55 and #56 which can proceed after this PR lands.

Reply:
I've already made a PR fixing [#56](https://github.com/nodejs/gyp-next/issues/56).